### PR TITLE
data: BenceDv

### PR DESCRIPTION
**Pull request by BenceDv**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5691"></a>

### CODE REVIEW

File: data/BenceDv.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ {
+      "name": "Twitter",
+      "url": "https://twitter.com/BenceDvorszki",
+      "icon": "FaTwitter"
+    },
+{
+      "name": "Discord",
+      "url": "https://discord.gg/CX47T3p96X",
+      "icon": "FaDiscord"
+    },
```
Comment (khushalarora) [inline data/BenceDv.json:26]: Please add a  square bracket &nbsp; **' ] '** &nbsp;&nbsp; between &nbsp; " } " &nbsp; and  &nbsp; " , "
```Suggestion
    }],
```

File: data/BenceDv.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  {
+    "title": "Scheduled to finish A levels",
+    "date": "July 2023",
+    "icon": "FaFlagCheckered",
+    "color": "grey",
+    "description": "From here the opportunities are endless, next destination: Florida",
+    "url": "https://goo.gl/maps/UueiqgmWuPRM9a2u9"
+}  
+  ]
+]
```
Comment (khushalarora) [inline data/BenceDv.json:38]: Delete this **' square bracket  ] '** :+1: 
```Suggestion
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hi @BenceDv, thank you for contributing :tada:

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**